cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocRuntimeJobManager.java

```diff
@@ -183,8 +185,10 @@ public void launch(RuntimeJobInfo runtimeJobInfo) throws Exception {
     LOG.debug("Launching run {} with following configurations: cluster {}, project {}, region {}, bucket {}.",
               runInfo.getRun(), clusterName, projectId, region, bucket);
 
-    // TODO: CDAP-16408 use fixed directory for caching twill, application, artifact jars
```

Comment:
Make sure to include this bug in the PR, and close it.

Reply:
The bug mentioned in the comment is already closed. Hence, removed the `TODO`.